File: baekjoon/18408/Main.c
#include <stdio.h>

int main()
{
    int a,b,c;
    int countA = 0;
    int countB = 0;

    scanf("%d%d%d",&a,&b,&c);

    if(a == 1){
        countA++;
    }else{
        countB++;
    }
    if(b == 1){
        countA++;
    }else{
        countB++;
    }
    if(c == 1){
        countA++;
    }else{
        countB++;
    }

    if(countA>countB){
        printf("1");
    }else{
        printf("2");
    }

        
}

File: baekjoon/18408/Main.java
import java.util.*;
import java.io.*;

public class Main{

  static int N;
  static int[] page_arr = new int[10];
  static int multiply = 1;
  public static void main(String[] args) throws IOException{

    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    StringBuilder sb = new StringBuilder();

    StringTokenizer st = new StringTokenizer(br.readLine(), " ");
    
    int A = Integer.parseInt(st.nextToken());
    int B = Integer.parseInt(st.nextToken());
    int C = Integer.parseInt(st.nextToken());

    int countA = 0;
    int countB = 0;

    if(A == 1){
      countA++;
    }else{
      countB++;
    }

    if(B == 1){
      countA++;
    }else{
      countB++;
    }

    if(C == 1){
      countA++;
    }else{
      countB++;
    }

    if(countA>countB){
      System.out.println(1);

    }else{
      System.out.println(2);
    }

  }
}